### DISCUSSION TITLE
Usage with any MOSP instance

### DISCUSSION THREAD

**Discussion by luke-on-security** [NC3-LU/PyMOSP#3] [Q&A] [answered]
Is it possible to use this with a random instance pleaz ?

**Comment by cedricbonhomme** [comment, marked as answer]
this is configurable, yes.
See [here](https://github.com/CASES-LU/PyMOSP/blob/master/tests/test_object.py#L12=) please.
Not all endpoints of the MOSP API are covered but almost all you will need for a normal scenario (which means creation, edition, deletion of objects).
